python/across_api/base/database.py

```diff
@@ -0,0 +1,13 @@
+import aioboto3  # type: ignore[import]
+from arc._lib import use_aws, get_ports  # type: ignore[import]
+
+
+def dynamodb_resource():
+    dynamodb_session = aioboto3.Session()
+    if use_aws():
+        return dynamodb_session.resource("dynamodb")
+    else:
+        port = get_ports()["tables"]
+        return dynamodb_session.resource(
+            "dynamodb", endpoint_url=f"http://localhost:{port}", region_name="us-east-1"
```

Comment:
There ought to be a way to get the region programmatically and to not hard-code it.

Reply:
It doesn't really matter for this because it's accessing the DynamoDB instance running locally here. However, I changed this to fetch from the `AWS_REGION_NAME` environment variable or default to `us-east-1`, which is how `architect-functions` does it (except they default to `us-west-1`).